Makefile

```diff
@@ -2,7 +2,15 @@
 
 all: deps build-client
 
-deps: clean deps-server optimize autoload deps-client
+deps: clean deps-server optimize autoload deps-client gems
+
+deps-production: clean deps-server optimize autoload deps-client gems-production
```

Comment:
In deps-production you're running gems-production even if gems hasn't been run?  Maybe I'm missing something, but if you ran `deps-production` before running `deps` then this would fail.